packages/taco/test/conditions/compound-condition.test.ts

```diff
@@ -205,4 +216,67 @@ describe('validation', () => {
     });
     expect(condition.value.conditionType).toEqual(CompoundConditionType);
   });
+
+  it('rejects invalid operator', () => {
+    const badObj = {
+      operator: 'invalid-operator',
+      operands: [testContractConditionObj, testTimeConditionObj],
+    };
+    expect(() => new CompoundCondition(badObj)).toThrow();
+  });
+
+  it.each(['or', 'and'])(
```

Comment:
Should `not` also be in here - it should reject empty operands as well?
```suggestion
  it.each(['or', 'and', 'not'])(
```